spec/models/review_spec.rb

```diff
@@ -8,11 +8,14 @@
   it { should respond_to :rating }
   it { should validate_presence_of :content }
   it { should validate_presence_of :rating }
+  it { should validate_presence_of :video }
 
-  context 'for rating values between 0 and 5' do
+  let(:video) { video = Fabricate(:video) }
+
+  context 'for rating values between 1 and 5' do
```

Comment:
https://github.com/thoughtbot/shoulda-matchers/blob/master/lib/shoulda/matchers/active_model/validate_inclusion_of_matcher.rb#L22

also you are using almost 1 year old version of shoulda-matchers might be that.